scripts/vote_2025_05_21.py

```diff
@@ -0,0 +1,285 @@
+8. Change initialSlashingAmountPWei from 1000 to 8 on Oracle Report Sanity Checker `0x6232397ebac4f5772e53285b26c47914e9461e75` 
+9. Revoke INITIAL_SLASHING_AND_PENALTIES_MANAGER_ROLE on Oracle Report Sanity Checker `0x6232397ebac4f5772e53285b26c47914e9461e75` from Aragon Agent `0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c`
+
+II. Add Easy Track setup for Managing MEV-Boost Relay Allowed List 
+10. Add `AddMEVBoostRelays` EVM script factory `0x00A3D6260f70b1660c8646Ef25D0820EFFd7bE60` to Easy Track
+11. Add `RemoveMEVBoostRelays` EVM script factory `0x9721c0f77E3Ea40eD592B9DCf3032DaF269c0306` to Easy Track 
+12. Add `EditMEVBoostRelays` EVM script factory `0x6b7863f2c7dEE99D3b744fDAEDbEB1aeCC025535` to Easy Track
+13. Set Easy Track's EVM Script Executor `0xFE5986E06210aC1eCC1aDCafc0cc7f8D63B3F977` as manager on the MEV-Boost Relay Allowed List `0xF95f069F9AD107938F6ba802a3da87892298610E`
+
+III. CSM: Reduce keyRemovalCharge
+14. Grant MODULE_MANAGER_ROLE on CSModule `0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F` to Aragon Agent `0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c`
+15. Reduce keyRemovalCharge from 0.05 to 0.02 ETH on CS Module `0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F`
+16. Revoke MODULE_MANAGER_ROLE on CSModule `0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F` from Aragon Agent `0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c`
+
+IV. Change Easy Track limits for Liquidity Observation Lab (LOL)
+17. Increase the limit from 2,100 to 6,000 stETH and extend the duration from 3 to 6 months on LOL AllowedRecipientsRegistry `0x48c4929630099b217136b64089E8543dB0E5163a`
+"""
+
+import time
+from typing import Dict
+from brownie import interface
+from brownie.network.transaction import TransactionReceipt
+from utils.agent import agent_forward
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.ipfs import calculate_vote_ipfs_description, upload_vote_ipfs_description
+from utils.config import (
+    contracts,
+    get_deployer_account,
+    get_is_live,
+    get_priority_fee,
+    EASYTRACK_EVMSCRIPT_EXECUTOR,
+    EASYTRACK_MEV_BOOST_ADD_RELAYS_FACTORY,
+    EASYTRACK_MEV_BOOST_REMOVE_RELAYS_FACTORY,
+    EASYTRACK_MEV_BOOST_EDIT_RELAYS_FACTORY,
+)
+from utils.easy_track import (
+    add_evmscript_factory,
+    create_permissions,
+)
+
+from utils.permissions import encode_oz_grant_role, encode_oz_revoke_role
+
+# Oracle sanity checker params
+NEW_INITIAL_SLASHING_AMOUNT_PWEI = 8
+UNCHANGED_INACTIVITY_PENATIES_AMOUNT_PWEI = 101
+NEW_EXITED_VALIDATORS_PER_DAY_LIMIT = 3600
+NEW_APPEARED_VALIDATORS_PER_DAY_LIMIT = 1800
+
+NEW_KEY_REMOVAL_CHARGE = 0.02 * 1e18
+
+NEW_LOL_LIMIT = 6000 * 1e18  # stETH
+NEW_LOL_PERIOD = 6  # months
+
+DESCRIPTION = """Contains separate updates approved by Lido DAO via Snapshot voting:
+
+1. **Post-Pectra update:** adjust Oracle Report Sanity Checker parameters to align with reduced slashing penalty and updated validator churn limits. Items 1-9.
```

Comment:
and updated validator churn limits - is this right? @kate-aleksseeva please check it